src/types.ts

```diff
@@ -28,3 +34,23 @@ export interface ProductCardProps {
 export interface SliderComponentProps {
   breed: BreedType[];
 }
+
+export interface SingleBreedSpecsProps {
+  breed: BreedType;
+  breedsSpecs:
+    | [
+        string,
```

Comment:
difficult to read structure. 

try to break this type into several subtypes (at least move this 

string
          | number
          | Length
          | {
              id: string;
              width: number;
              height: number;
              url: string;
            }
            
 into a separate type)